tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,4 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result is False
```

Comment:
You could make this test more robust by verifying that the items in the unchanged inventory are actually the items we started with (and not some other 3 items) and also check that inventory list lengths are unchanged
```py
    assert item_a in fatimah.inventory
    assert item_b in fatimah.inventory
    assert item_c in fatimah.inventory
    assert len(jolie.inventory) == 0
    assert len(fatimah.inventory) == 3
```